index.js

```diff
@@ -1,155 +1,103 @@
-// Variables to call Elements by ID.
-const showFormBtn = document.getElementById("showFormBtn"),
-    form = document.getElementById("bookForm"),
-    libraryDivContainer = document.getElementById("renderLibrary"),
-    removeBookBtnQuery = document.getElementsByClassName("deleteBookBtn"),
-    openModalTrigger = document.querySelector(".trigger"),
-    closeModalTrigger = document.querySelector(".close"),
-    modal = document.querySelector(".modal")
-
-// Initialize the Library Array and new book variable.
-let myLibrary = []
-let newBook
-let tempLibrary
-
-// Book Constructor:
-function Book(title, author, pages, read = false) {
-    this.title = title,
-    this.author = author,
-    this.pages = pages,
-    this.read = read
-}
-
-// Modal EventListeners for Open, Close, and click outside Modal.
-openModalTrigger.addEventListener("click", () => modal.classList.add("open"))
-closeModalTrigger.addEventListener("click", () => modal.classList.remove("open"))
-window.addEventListener("click", (event) => {
-    if(event.target === modal) {
-        modal.classList.remove("open")
+const myLibrary = [];
+const booksContainer = document.getElementById('books-container');
+const formSubmit = document.querySelector('#addBookForm button');
+const addBookForm = document.getElementById('addBookForm');
+
+const Book = (title, author, pages, read, id = Date.now()) => ({
+  title, author, pages, read, id,
+});
+// function Book(title, author, pages, read, id = Date.now()) {
+//   this.title = title;
+//   this.author = author;
+//   this.pages = pages;
+//   this.read = read;
+//   this.id = id;
+// }
```

Comment:
## Remove commented out code 
- [ ] Great job having achieved this constructor functionality bypassing with an arrow function, bypassing the traditional way of doing it. We however to take care of the commented out code. It makes the codebase ugly. It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
- [ ] Also the commit messages are not that informing of the changes that got carried out in this commit. In your next round of changes be sure to commit as often and ensure the commit messages are as descriptive as the changes they introduce. One giant piece of an overhaul in the codebase confuses the code reviewer of what exactly informed the decision to refactor the code as in this instance. The commit messages should act as a guide for a reviewer to track changes made in the codebase. Ensure each commit is doing one purpose only at a time. Thank you!